database.sql

```diff
@@ -0,0 +1,25 @@
+CREATE TABLE Users (
+    userId SERIAL PRIMARY KEY,
+    username VARCHAR(100),
+    email VARCHAR(100),
+    passwd  VARCHAR(200),
+    team VARCHAR(200)
+);
+
+CREATE TABLE Team (
+    teamId SERIAL PRIMARY KEY,
+    listUsers VARCHAR(100),
```

Comment:
To create a relation we don't need to listUsers we can do a join.

https://hasura.io/learn/database/postgresql/core-concepts/6-postgresql-relationships/